tox.ini

```diff
@@ -29,16 +28,14 @@ commands =
 [testenv:test-exe]
 basepython = python3.7
 deps =
-    -r{toxinidir}/requirements.txt
-    {toxinidir}
+    -r{toxinidir}/requirements-build.txt
```

Comment:
Sorry did not notice this now till I tried it, but I think we need to add the lines from the `make-exe` script to the `test-exe` script:
```
run('pip install -r requirements.txt')
run('pip install .')
```
Specifically, we rely on `nose` to run the smoke tests and still import from the CLI/botocore codebase (even though we should probably update that in the future to require no dependencies)

Reply:
Actually looking at both of these. I say we make the two changes:

* Do not make any changes to the `test-exe` environment.
* Do not specify `deps` for the `sign-exe`. Looking that one over I don't think it has any depdencies.

Given that the entire dependency closure for `requirements.txt` has stayed the same, I would not worry too much about these other environments. We just want to make sure the exe's can be built offline.